ktor-client/ktor-client-features/ktor-client-json/ktor-client-json-jvm/test/io/ktor/client/features/json/test/JsonTest.kt

```diff
@@ -13,16 +14,29 @@ import io.ktor.response.*
 import io.ktor.routing.*
 import io.ktor.server.engine.*
 import io.ktor.server.jetty.*
+import kotlinx.serialization.Serializable
+import java.util.*
 import kotlin.test.*
 
+@Serializable
 data class Widget(
-    val name: String,
-    val value: Int,
-    val tags: List<String> = emptyList()
+        val name: String,
```

Comment:
Could you use the kotlin style guide(https://kotlinlang.org/docs/reference/coding-conventions.html)?
It could be configured as default style guide in IDEA `Code Style` section for kotlin.

Reply:
Thanks